lib/pod/command/why.rb

```diff
@@ -134,6 +134,7 @@ def all_paths(source, target, graph)
         def search(source, target, graph, all_paths)
           return all_paths[source] if all_paths.key?(source)
           return [[target]] if source == target
+
```

Comment:
Added this to fix a RuboCop violation

https://docs.rubocop.org/rubocop/cops_layout.html#layoutemptylineafterguardclause